Formula/makepkg.rb

```diff
@@ -14,14 +14,12 @@ class Makepkg < Formula
     sha256 high_sierra: "c8f2f6999669c56b5e40e2608ad1e0adfe2c8eb73f8cef959a229856d21da6ed"
   end
 
-  disable! date: "2022-03-28", because: "depends on fakeroot which does not build"
-
   depends_on "asciidoc" => :build
```

Comment:
Is this still needed if we're not building docs?

Reply:
Probably not - I will check this before the next push.

Reply:
Also, we might be able to drop `docbook-xsl` here as well. I will update this whatever can be dropped.